src/StackExchange.Redis/ServerEndPoint.cs

```diff
@@ -567,7 +567,7 @@ internal string GetProfile()
 
         internal string? GetStormLog(Message message) => GetBridge(message)?.GetStormLog();
 
-        internal Message GetTracerMessage(bool assertIdentity)
+        internal Message GetTracerMessage(bool checkResponse)
```

Comment:
Do we need to pass `checkResponse: true` when calling this from `KeepAlive()`?

Reply:
Discussed on call here - effectively checking for PONG with the same rigor/fail so we're approx 1:1 with the Multiplexer.UniqueId (which doesn't change) check in terms of fidelity.